core_backend/app/schemas.py

```diff
@@ -9,3 +11,16 @@ class FeedbackSentiment(str, Enum):
     POSITIVE = "positive"
     NEGATIVE = "negative"
     UNKNOWN = "unknown"
+
+
+class QuerySearchResult(BaseModel):
```

Comment:
Given the circular issues we are experimenting with feedback should we create a feedback module?

Reply:
You might be right. Though breaking it up may cause other circular import issues. We should consider a refactor for sure though. That module is becoming too big.

Reply:
But in a future PR!